ledger/simulation/simulator.go

```diff
@@ -165,10 +141,9 @@ func (s Simulator) check(hdr bookkeeping.BlockHeader, txgroup []transactions.Sig
 	// Verify the signed transactions are well-formed and have valid signatures
 	_, err = verify.TxnGroupWithTracer(txnsToVerify, &hdr, nil, s.ledger, debugger)
```

Comment:
so missing-signatures is only if the signature is literally not specified, not whether it is valid

Reply:
Correct. I went into further detail here: https://github.com/algorand/go-algorand/pull/4439#discussion_r1113415395